icons/milestone.svg

```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M18 6H5a2 2 0 0 0-2 2v3a2 2 0 0 0 2 2h13l4-3.5L18 6Z" />
   <path d="M12 13v8" />
   <path d="M12 3v3" />
+  <path d="M17.749 13a2 2 0 0 0 1.317-.495l2.574-2.252a1 1 0 0 0 0-1.506l-2.575-2.252A2 2 0 0 0 17.749 6H5a2 2 0 0 0-2 2v3a2 2 0 0 0 2 2z" />
```

Comment:
2px on the left side feels a bit too extreme, it's not in line with our guidelines either:
```suggestion
  <path d="M4 6a1 1 0 0 0-1 1v5a1 1 0 0 0 1 1h13a2 2 0 0 0 1.152-.365l3.424-2.317a1 1 0 0 0 0-1.635l-3.424-2.318A2 2 0 0 0 17 6z" />
```

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M12+13v8%22+/%3E%3Cpath+d=%22M12+3v3%22+/%3E%3Cpath+d=%22M4+6a1+1+0+0+0-1+1v5a1+1+0+0+0+1+1h13a2+2+0+0+0+1.152-.365l3.424-2.317a1+1+0+0+0+0-1.635l-3.424-2.318A2+2+0+0+0+17+6z%22+/%3E&name=image"><img alt="icons" width="400" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTIgMTN2OCIgLz4KICA8cGF0aCBkPSJNMTIgM3YzIiAvPgogIDxwYXRoIGQ9Ik00IDZhMSAxIDAgMCAwLTEgMXY1YTEgMSAwIDAgMCAxIDFoMTNhMiAyIDAgMCAwIDEuMTUyLS4zNjVsMy40MjQtMi4zMTdhMSAxIDAgMCAwIDAtMS42MzVsLTMuNDI0LTIuMzE4QTIgMiAwIDAgMCAxNyA2eiIgLz4KPC9zdmc+.svg"/><br/>Open lucide studio</a>